bedrock/firefox/templates/firefox/family/includes/modules/public-wifi.html

```diff
@@ -17,14 +17,12 @@ <h3 class="c-subtitle">An extra sloppy scoop of public wifi</h3>
       </div>
 
       <div class="l-container t-scroll-snap">
```

Comment:
@reemhamz there are styles associated with `t-scroll-snap` that look strange when we only have the one card, let's remove those styles